src/styles/blocks/cards-container.scss

```diff
@@ -0,0 +1,19 @@
+.cards-container {
+  padding: $cards-container-py $cards-container-px;
+  width: getCardsContainerWidth(
+    4,
+    $card-width,
+    $cards-column-gap,
+    $cards-container-px
+  );
+  display: flex;
+  flex-flow: row wrap;
```

Comment:
row is the value by default, as for me, it would be better to use `flex-wrap: wrap`

Reply:
agree, `flex-wrap: wrap` should work here

Reply:
haven't fixed